slo_testing/hg/hg_testing_README.md

```diff
@@ -0,0 +1,105 @@
+# How to Test the SLO Terraform Provider - Hosted Grafana
```

Comment:
everything within `slo_testing` is to be deleted, left in for now for testing purposes.